tests/unit/query_result_tests.py

```diff
@@ -428,21 +428,11 @@ def test_iteration_with_invalid_options(self):
         Test that iteration raises an exception when "skip" and/or "limit" are
```

Comment:
Need to fix this comment now that tests with skip are removed.

Reply:
you're right!